lightning-background-processor/src/lib.rs

```diff
@@ -291,6 +301,10 @@ impl BackgroundProcessor {
 			if let Some(ref handler) = net_graph_msg_handler {
 				persister.persist_graph(handler.network_graph())?;
 			}
+
+			// Persist Scorer on exit
+			persister.persist_scorer(&scorer)?;
+
 			Ok(())
```

Comment:
```suggestion
			// Persist Scorer on exit
			persister.persist_scorer(&scorer)?;

			Ok(())
```

Why the failure is ignored and reported `Ok()` any way? not able to catch this, sorry